### DISCUSSION TITLE
Elk toot/pouet with multiple answers : I can see only my reply

### DISCUSSION THREAD

**Discussion by boly38** [elk-zone/elk#2687] [Q&A] [answered]
Hi
I search a little in issue/discussion without success so I create this thread.

I'm using ELK and for some pouet I got a number of answers > 0 where simply I can't see all-of-them 

Exemple
https://elk.zone/octodon.social/@mhucka@fediscience.org/112089726298231678

on this pouet there is **3** answers

I can see mine, but to show the other I do need to open another mastodon client 

Exemple
https://fediscience.org/@mhucka/112089726221716169

Is this a bug ? 

In advance thanks
PS: I like your client so much, especially for the "translate" feature !

**Comment by userquin** [comment]
Replies number comes from `octodon.social`, `replies_count` is 3, Elk just displays the replies using  `replies_count`:

![imagen](https://github.com/elk-zone/elk/assets/6311119/56072761-3655-421a-b6e6-1d2692c414d5)

**Reply by userquin** [reply]
Here the context for the toot (only 2 descendants):

![imagen](https://github.com/elk-zone/elk/assets/6311119/522e72ef-31a4-4187-b080-5c7a79c1cb2b)

**Reply by boly38** [reply]
thanks for your response
- 3 responses: the digit is correct
- context: only 2 descendants: ok thats mine
- what I didnt get is why 3 responses didnt trigger 3 different sections under the toot like on [this view](https://fediscience.org/@mhucka/112089726221716169) : having WikiPathways + anlomedad in addition to me 

Maybe you mean that octodon.social has this issue in his scope? ie. not providing this toot in context
it's true that going to this view give me same (unexpected) result : https://octodon.social/@mhucka@fediscience.org/112089726298231678

(I send a poke to the [octodon.social instance](https://octodon.social/about) admin to get his review : version used: v4.3.0-alpha.3+glitch )

**Reply by userquin** [reply]
It seems something related to octodon, NOT LISTED, getting the status uri (https://fediscience.org/users/mhucka/statuses/112089726221716169) from the toot:

![20240314_164223](https://github.com/elk-zone/elk/assets/6311119/10f44d4d-56b2-4db1-a1c3-6341c653db31)

**Reply by userquin** [reply]
I'm going to check the logic, that uri not being used, in the original we have 4 descendants:

![20240314_165210](https://github.com/elk-zone/elk/assets/6311119/0ddbb0e1-30e4-4902-9ca8-fa256d451a30)

**Comment by boly38** [comment, marked as answer]
I got some details from admin- of octodon.social instance- (cf under),
and this explain well the current behavior. 

Thanks to him and to you @userquin for your contribution to help me to understand anyway 🙏 

```
it's the expected behavior; the local instance filters replies using its own blocklist 
as it would have to pull them from their original servers, which are (partially) blocked.
(...)
you won't see their posts unless you explicitly follow them.
our server still reports the original count as established by their home server,
so if you see a different number you can go to the source post on its own server,
 and see all replies that they have not blocked themselves. they just won't all be relayed by our server.
optionally, elk could fetch those replies, but that's a specific feature that not everyone might want, 
and it comes with new privacy expectations.
```

**Reply by boly38** [reply]
The mastodon clients may produce hint in case of context 
direct child count is less than replies count 
This to explain that some context may have been filtred by current instance

**Reply by userquin** [reply]
iirc there are some places in Elk where we add some hints: maybe we can add some flag if there is a missmatch between the replies_count and the descendants length... feel free to file an issue requesting that feature.

**Reply by boly38** [reply]
added : #2688
